bin/deploy

```diff
@@ -52,8 +52,8 @@ done
 
 echo "Deploying $NETWORK_NAME network...";
 
-# TODO: this fails without an error if not at the HEAD of a given branch
-BRANCH=`git symbolic-ref --short -q HEAD`
+# TODO: this fails if not at the HEAD of a given branch
```

Comment:
This PR is fine as-is (approved), but what is the TODO left here? Do you want it to not fail if not on a named branch? That would be easy to fix as you just disable errors and re-enable after the command.

One method:

```
set +e
git command here
set -e
```

Maybe cleaner alternative:

```
git command here || true
```

You could restore the `--quiet` option at this point and avoid the ugly error message. Sorry I didn't recognize this at the time, wasn't sure what the problem or goal was.

Reply:
It's fine, I think it is good to fail with an error at least so I know where to look if I ever end up in this situation again. I don't think we should just bypass setting the variable, it's needed for the subsequent check.